index.js

```diff
@@ -297,6 +298,48 @@ Blockchain.prototype.getBlock = function (hash, cb) {
 }
 
 /**
+ *Gets a block by its hash
+ * @method getBlocks
+ * @param blockId - the block's hash or number
+ * @param {Function} cb - the callback function
+ */
+Blockchain.prototype.getBlocks = function (blockId, maxBlocks, skip, reverse, cb) {
+  
+  var self = this
+  var blocks = []
+  var i = -1
+
+  function nextBlock(blockId){
+
+    self.getBlock(blockId, function(err, block){
+
+      i++
+
+      if(err){
+        return cb(null, blocks)
+      }
+
+      var nextBlockNumber = ethUtil.bufferToInt(block.header.number) + (reverse ? -1 : 1)
+      
+      if(i !== 0 && skip && i % (skip + 1) !== 0)
+        return nextBlock(nextBlockNumber)
+    
+      blocks.push(block)
+
+      if(blocks.length === maxBlocks)
+        return cb(null, blocks)
```

Comment:
I don't think this is allowed by the lint. But I might be wrong. can you run `npm run lint`?